admin/src/collections/Recipients.tsx

```diff
@@ -1,16 +1,16 @@
 import {
```

Comment:
nit: .ts instead of tsx?

Reply:
It has to be `.tsx`. The `AsyncPreviewComponent` uses JSX syntax.